### ISSUE TITLE
Screen position out of view frustum

### ISSUE THREAD

**Issue by JaviAibar** [yasirkula/UnityBezierSolution#25]
**Description of the bug**

Changing camera position to the one given by FindNearestPointTo then the exception "Screen position out of view frustum" is raised.

**Reproduction steps**

1. Activate property loop.
2. Call the method FindNearestPointTo
3. Position of the camera to returned position

**Platform specs**

- Unity version: 2020.3.11f1
- Platform: Windows 10 Pro versión 21H1
- Device: In editor, in play mode
- How did you download the plugin: Asset Store
- Plugin version: 2.2.0

**Comment by yasirkula** [comment]
Can you Debug.Log the returned point and see what it prints? Is it _NaN_? If so, can you create a prefab from the spline and the point you call FindNearestPointTo with and make a unitypackage with these prefabs so that I can reproduce the issue?

**Comment by JaviAibar** [comment]
I found the problem!

I've made some prints as you suggested and everything was ok. The thing is that the camera is attached to the player and the speed of the player's rigidbody was being constantly multiplyied in the update to some massive numbers. Therefore when the camera was tried to be rendered, it crashes horribly, so it didn't have anything to make with beziers.

I'm so sorry for the annoyances and thanks for the support.
Wish you the best!
